rpc/src/v1/impls/parity.rs

```diff
@@ -417,6 +510,29 @@ impl<C, M, U, S> Parity for ParityClient<C, M, U> where
 		}))
 	}
 
+		fn raw_block_by_number(&self, num: BlockNumber, include_txs: bool) -> BoxFuture<Option<RichBlock>> {
```

Comment:
As commented above, the type of this function should be `fn raw_block_by_number(&self, num: BlockNumber, include_txs: bool) -> BoxFuture<Option<Bytes>>`.

Reply:
Should not have the `include_txs` parameter.

Reply:
@andresilva did you mean that I should convert the result of this function into Bytes using .rlp_bytes here first, instead of in fn submit_block (in the same file)? 

Also, to clarify on the include_txs parameter, since this function is to be used to allow miners to make any changes to the pending block before submission, is it then the case that I should always assume we include localized transactions and set `transactions` of the block to 

BlockTransactions::Full(block.view().localized_transactions().into_iter().map(|t| Transaction::from_localized(t)).collect()) ?